src/AppInstallerCommonCore/Fonts.cpp

```diff
@@ -172,4 +840,120 @@ namespace AppInstaller::Fonts
         fontFamily.Faces = std::move(fontFaces);
         return fontFamily;
     }
+
+    std::wstring GetFontFileTitle(const std::filesystem::path& fontFilePath)
+    {
+        // This code can fail in a number of ways, including if the file isn't actually
+        // a font file or does not have a title.
+        wil::com_ptr<IPropertyStore> pPropertyStore;
+        THROW_IF_FAILED(SHGetPropertyStoreFromParsingName(fontFilePath.c_str(), nullptr, GPS_DEFAULT, IID_PPV_ARGS(&pPropertyStore)));
+        PROPVARIANT prop;
+        PropVariantInit(&prop);
+        THROW_IF_FAILED(pPropertyStore->GetValue(PKEY_Title, &prop));
+        std::wstring title;
+        if (prop.pwszVal)
+        {
+            title = prop.pwszVal;
+        }
+        THROW_IF_FAILED(PropVariantClear(&prop));
+        return title;
+    }
+
+    // This will create an inventory of all known permanently installed fonts to the user.
+    // A font is "permanently installed" if it is present in the Font registry for the machine or the user.
+    // This will not include fonts that are temporarily installed for the session.
+    std::vector<FontFileInfo> GetInstalledFontFiles()
+    {
+        auto fontFiles = std::vector<FontFileInfo>();
+
+        try
+        {
+            // Iterate through scopes for machine and user
+            std::vector<Manifest::ScopeEnum> scopes = { Manifest::ScopeEnum::Machine, Manifest::ScopeEnum::User };
+            for (const auto& scope : scopes)
```

Comment:
```suggestion
            for (const auto& scope : { Manifest::ScopeEnum::Machine, Manifest::ScopeEnum::User })
```
Removes the unnecessary allocation of a vector.